packages/react/src/themes/teams/components/Loader/loaderStyles.ts

```diff
@@ -48,15 +47,18 @@ export default {
       position: 'relative',
     }
     const svgAnimation: ICSSInJSStyle = {
-      animationName: t.renderer.renderKeyframe(
-        () =>
+      animationName: {
+        keyframe: ({ from, to }) =>
           ({
             to: {
-              transform: `translate3d(0, ${v.svgTranslatePosition[p.size]}, 0)`,
+              transform: `translate3d(, ${v.svgTranslatePosition[p.size]}, 0)`,
```

Comment:
just to be sure - is `0` removed intentionally?

Reply:
No, I reverted this change.